specificatie/naam.yaml

```diff
@@ -0,0 +1,183 @@
+openapi: 3.0.3
+info:
+  title: Naam definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Aanhef:
+      type: string
+      description: |
+        Kun je gebruiken als aanhef in een brief gericht aan persoon.
+      example: "Hoogwelgeboren heer"
+    Aanschrijfwijze:
+      type: string
+      description: |
+        Samengestelde naam die je kunt gebruiken in de communicatie met de persoon.
+      example: "H.W. baron van den Aedel"
+    GebruikInLopendeTekst:
+      type: string
+      description: |
+        Naam van persoon die je kunt gebruiken als je in lopende tekst (bijvoorbeeld in een brief) aan persoon refereert.
+      example: "baron Van den Aedel"
+    Geslachtsnaam:
+      type: string
+      maxLength: 200
+      description: |
+        De achternaam van een persoon.
+      example: "Vries"
+    RegelVoorafgaandAanAanschrijfwijze:
+      type: string
+      description: |
+        Deze regel moet als aparte regel boven de aanschrijfwijze worden geplaatst. Komt alleen voor bij personen met een adellijke titel of predicaat.
+      example: "De hoogwelgeboren heer"
+    Voorletters:
+      type: string
+      description: |
+        De voorletters van de persoon, afgeleid van de voornamen.
+      example: "P.J."
+    Voornamen:
+      type: string
+      maxLength: 200
+      description: |
+        De verzameling namen voor de geslachtsnaam, gescheiden door spaties.
+    Voorvoegsel:
+      type: string
+      maxLength: 10
+      example: "de"
+    AbstractNaamBasis:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaNaam: '#/components/schemas/GbaNaamBasis'
+          Naam: '#/components/schemas/NaamBasis'
+    AbstractNaam:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaNaam: '#/components/schemas/GbaNaam'
+          Naam: '#/components/schemas/Naam'
+    GbaNaamBasis:
+      type: object
+      properties:
+        voornamen:
+          $ref: '#/components/schemas/Voornamen'
+        voorvoegsel:
+          $ref: '#/components/schemas/Voorvoegsel'
+        geslachtsnaam:
+          $ref: '#/components/schemas/Geslachtsnaam'
+        adellijkeTitelPredikaat:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+    NaamBasis:
+      allOf:
+        - $ref: '#/components/schemas/GbaNaamBasis'
+        - type: object
+          properties:
+            voorletters:
```

Comment:
kan je hier volledigeNaam  aan toevoegen? Wellicht nog andere constructie, zodat deze wel in persoonBeperkt en Persoon zit, niet in ouders, partners, kinderen. @CathyDingemanse of wil je dat ook in ouders, partners, kinderen toegevoegd hebben?

Reply:
done. Het is nog niet aangepast voor gerelateerde personen

Reply:
> 
Nee, gegevens van ouders, kinderen en partners zijn nu eigenschappen van de persoon. Ik heb nog geen vraag gehad om ook daar over de volledige naam van gerelateerden te kunnen beschikken.